packages/taquito/src/batch/rpc-batch-provider.ts

```diff
@@ -72,6 +76,20 @@ export class OperationBatch extends OperationEmitter {
     return this;
   }
 
+  /**
+   *
+   * @description Transfer tickets from tx-rollup-L2-address to an L1-address
```

Comment:
I am not sure the description is accurate. Should it be "Transfer tickets from a Tezos address (tz1,tz2 or tz3) to a smart contract address( KT1)" ?